test/StandardSqlFormatterTest.js

```diff
@@ -358,8 +358,7 @@ describe("StandardSqlFormatter", function() {
             "SELECT\n" +
             "  a\n" +
             "FROM\n" +
-            "  b --comment\n" +
-            ";"
+            "  b --comment;"
```

Comment:
This is a bug. Semicolon after a comment should not be moved to the end of the comment.

You should be able to fix this by calling `trimTrailingWhitespace` instead of `trimEnd` inside the new `formatQuerySeparator` function.

Reply:
@nene 
Thanks for the review. I made the adjustments you asked and the test now is behaving like it was before.